### PR TITLE
[MM-28200] Cloud Billing - Admin Console sections and layout

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
First bit of the Admin Console pages for Cloud Billing
No functionality yet, just the setup of the `admin_definition` and wrapper components.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28200

#### Related PRs
Redux: https://github.com/mattermost/mattermost-redux/pull/1227

### CODE REVIEW

File: components/admin_console/billing/billing_history.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import React from 'react';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+type Props = {
+
+};
+
+const BillingHistory: React.FC<Props> = () => {
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_history.tsx:12]: You can leave out the `Props` if there aren't any.

```suggestion
const BillingHistory: React.FC = () => {
```

File: components/admin_console/billing/billing_subscriptions.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.BillingSubscriptions {
+    .BillingSubscriptions__topWrapper {
+        display: flex;
+    }
+}
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_subscriptions.scss:5]: nit: empty line

File: components/admin_console/billing/billing_history.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <div className='wrapper--fixed BillingHistory'>
+            <FormattedAdminHeader
+                id='admin.billing.history.title'
+                defaultMessage='Billing History'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '463px'}}>
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_history.tsx:21]: should the styling be moved to its own scss file?

File: components/admin_console/billing/billing_subscriptions.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+
+import './billing_subscriptions.scss';
+
+type Props = {
+
+};
+
+const BillingSubscriptions: React.FC<Props> = () => {
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_subscriptions.tsx:14]: Recommend removing `Props` here if they're empty.

```suggestion
const BillingSubscriptions: React.FC = () => {
```

File: components/admin_console/billing/billing_subscriptions.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    return (
+        <div className='wrapper--fixed BillingSubscriptions'>
+            <FormattedAdminHeader
+                id='admin.billing.subscription.title'
+                defaultMessage='Subscriptions'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '81px', marginBottom: '20px'}}>
```
Comment (ali-farooq0) [inline components/admin_console/billing/billing_subscriptions.tsx:23]: Is styling done inline due to theming? if not, then I would recommend moving this to the scss file.

File: components/admin_console/billing/company_info.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import {FormattedMessage} from 'react-intl';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+import BlockableLink from 'components/admin_console/blockable_link';
+
+type Props = {
+
+};
+
+const CompanyInfo: React.FC<Props> = () => {
```
Comment (ali-farooq0) [inline components/admin_console/billing/company_info.tsx:14]:
```suggestion
const CompanyInfo: React.FC = () => {
```

File: components/admin_console/billing/company_info.tsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    return (
+        <div className='wrapper--fixed CompanyInfo'>
+            <FormattedAdminHeader
+                id='admin.billing.company_info.title'
+                defaultMessage='Company Information'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '432px'}}>
```
Comment (ali-farooq0) [inline components/admin_console/billing/company_info.tsx:23]: styling

File: components/admin_console/billing/company_info_edit.tsx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+import {FormattedMessage} from 'react-intl';
+
+import BlockableLink from 'components/admin_console/blockable_link';
+import SaveButton from 'components/save_button';
+
+type Props = {
+
+};
+
+const CompanyInfoEdit: React.FC<Props> = () => {
```
Comment (ali-farooq0) [inline components/admin_console/billing/company_info_edit.tsx:14]:
```suggestion
const CompanyInfoEdit: React.FC = () => {
```

File: components/admin_console/billing/company_info_edit.tsx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '432px'}}>
+                        {'Edit Company Info View'}
+                    </div>
+                </div>
+            </div>
+            <div className='admin-console-save'>
+                <SaveButton
+                    saving={false} // TODO
+                    // disabled={this.props.isDisabled || !this.state.saveNeeded || (this.canSave && !this.canSave())}
```
Comment (ali-farooq0) [inline components/admin_console/billing/company_info_edit.tsx:39]: should these be removed?

File: components/admin_console/billing/payment_info.tsx [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+import {FormattedMessage} from 'react-intl';
+
+import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
+import BlockableLink from 'components/admin_console/blockable_link';
+
+type Props = {
+
+};
+
+const PaymentInfo: React.FC<Props> = () => {
```
Comment (ali-farooq0) [inline components/admin_console/billing/payment_info.tsx:14]:
```suggestion
const PaymentInfo: React.FC = () => {
```

File: components/admin_console/billing/payment_info.tsx [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+    return (
+        <div className='wrapper--fixed PaymentInfo'>
+            <FormattedAdminHeader
+                id='admin.billing.payment_info.title'
+                defaultMessage='Payment Information'
+            />
+            <div className='admin-console__wrapper'>
+                <div className='admin-console__content'>
+                    <div style={{border: '1px solid #000', width: '100%', height: '81px', marginBottom: '20px'}}>
```
Comment (ali-farooq0) [inline components/admin_console/billing/payment_info.tsx:23]: styling

File: components/admin_console/billing/payment_info_edit.tsx [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+import {FormattedMessage} from 'react-intl';
+
+import BlockableLink from 'components/admin_console/blockable_link';
+import SaveButton from 'components/save_button';
+
+type Props = {
+
+};
+
+const PaymentInfoEdit: React.FC<Props> = () => {
```
Comment (ali-farooq0) [inline components/admin_console/billing/payment_info_edit.tsx:14]:
```suggestion
const PaymentInfoEdit: React.FC = () => {
```

File: package.json [hunk 13]
```diff
@@ -36,7 +36,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#97f0744b822f1114aa952115527ce8aae0bcc422",
+    "mattermost-redux": "github:mattermost/mattermost-redux#2c5ca00792fc5785366c4877275628de4aedb24c",
```
Comment (ali-farooq0) [inline package.json:39]: was this version bump intended?

File: components/admin_console/billing/billing_history.tsx (hunk 1 above)
Comment (devinbinnie) [reply components/admin_console/billing/billing_history.tsx:12]: Props will likely be added later, but I can remove them now if you want.

File: components/admin_console/billing/billing_history.tsx (hunk 3 above)
Comment (devinbinnie) [reply components/admin_console/billing/billing_history.tsx:21]: All the in-line styles are temporary to show where everything goes.

File: components/admin_console/billing/company_info_edit.tsx (hunk 9 above)
Comment (devinbinnie) [reply components/admin_console/billing/company_info_edit.tsx:39]: I might need them when the guts go in, currently there's nothing to save.

File: package.json (hunk 13 above)
Comment (devinbinnie) [reply package.json:39]: Yep, sorry should have included the related PR. Will add to description.

File: components/admin_console/billing/billing_history.tsx (hunk 1 above)
Comment (ali-farooq0) [reply components/admin_console/billing/billing_history.tsx:12]: ahh gotcha, seeing that this is WIP, then it's all good 👍

File: components/admin_console/billing/billing_history.tsx (hunk 3 above)
Comment (ali-farooq0) [reply components/admin_console/billing/billing_history.tsx:21]: Ahh gotcha.

**Pull request merged**